decentraland/unity-explorer
# fix: dclexperience: Octopus NPC dialog scale is huge

## What does this PR change?
Fix #705 

![image](https://github.com/decentraland/unity-explorer/assets/64659061/2f6b5652-9224-4117-9a39-e8e8601e4274)

## How to test the changes?
1. Launch the explorer,
2. Go to `dclexperience.dcl.eth` world (command: **/world dclexperience.dcl.eth**).
3. Talk with the octopus.
4. Observe the conversation UI is resized in order to adapt it to the current resolution (it shouldn't exist much difference in terms of dialog size).

**IMPORTANT NOTE TO TEST THIS FIX**: Lately, the `dclexperience.dcl.eth` world is not being loaded well sometimes, so please BE SURE that the dialog with the octopus advances when you click, if not, it does mean that the scene was not correctly loaded and you will have to go back to the point 2 and load the world again until it works correctly.

## Implementation notes
Finally, it was enough to change the `Scale Mode` of the `UIToolkit RootCanvas Settings` to make the UI's resize orderer by the scene works correctly! 👍 

![image](https://github.com/decentraland/unity-explorer/assets/64659061/269988a6-1333-45a2-ae0f-75f679b82a3f)

I realized that I was not able to test the fix well because most of the times the `dclexperience.dcl.eth` world was not being loaded correctly! ...and it was provoking an STOP on the scene flow, so none of the UI dialogs were receiving any resize from the scene.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
